src/main/java/com/cburch/logisim/fpga/data/MapComponent.java

```diff
@@ -651,6 +652,7 @@ public String getDisplayString(int pin) {
           ioConst |= constants.get(i) & 1;
         }
       }
+      if (pin < -1) return s.toString();
```

Comment:
can we have magic value "-2" replaced with constant? If not or it makes no sense then please add the comment about why it is used here.

Reply:
The magic value has been replaced by ONLY_IO_MAP_NAME.
Done